get-version.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+"""
+Get latest JanusGraph version from GitHub API. Optional argument can be used to specify the version prefix (e.g. 0.3).
+"""
+import sys, json, ssl
+if sys.version_info >= (3, 0):
+  from urllib import request
+  ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Yep I can add a comment. I'd prefer not to have the Python script at all. Do you think it would be okay to require [jq](https://stedolan.github.io/jq/) for building and testing the images?

Reply:
You only need the Python script so `update.sh` can be used to update to a new release version without needing any configuration at all, right?
Wouldn't it be already a good first step if `update.sh` would simply take the version as an argument? So, if we want to update to version `0.2.3`, then we just execute
```sh
./update.sh 0.2.3
git add .
git commit -s -m"Update to 0.2.3"
```

Updating to `0.3.2` should probably be a separate commit any way. So, I don't mind having to execute `./update.sh 0.3.2` for that again.

Reply:
Okay I'll think more on this. I would like to keep the automation but I also don't like the extra script. I was thinking about requiring jq as then a separate script probably wouldn't be needed.

Reply:
I'm not against requiring `jq`. I only wondered whether that isn't already too much automation, especially since someone will have to execute the `update.sh` script manually anyway.
But just having to execute the script which does everything without any configuration so the updated version can be committed directly also sounds nice. So, we can also just go ahead as you proposed.